samples/stock/demo/relative-rotation-graph/demo.html

```diff
@@ -0,0 +1,25 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/modules/series-label.js"></script>
+<script src="https://code.highcharts.com/modules/annotations.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<figure class="highcharts-figure">
+    <div id="container"></div>
+    <p class="highcharts-description">
+        Relative Rotation Graphs visualize the relation between the strength and
+        momentum of various assets against a chosen benchmark (i.e. S&P 500
+        index) and themselves. The performance is visualized in four quadrants.
+        Leading (green) - the asset is stronger than the benchmark and it's
+        getting stronger (positive momentum), Weakening (yellow) - the asset is
+        stronger than the benchmark, but it's getting weaker (negative
+        momentum), Lagging (red) - the asset is weaker than the benchmark and
+        it's getting weaker, Improving (blue) - the asset is weaker than the
+        benchmark, but it's getting stronger.
+        </p>
+        <br>
+
+        <a href="https://holdingbay.co.uk/blog/events/546-unlocking-the-power-of-relative-rotation-graphs/">
+        Read more about the RRG algorithm here.</a>
```

Comment:
I don't think we should redirect users to external sources, @TorsteinHonsi ? I know [Wikipedia](https://en.wikipedia.org/wiki/Relative_strength) has a big obscure description, but I would rather go with Wikipedia link